src/pkgcheck/checks/metadata.py

```diff
@@ -1363,13 +1382,18 @@ def __init__(self, *args, use_addon):
             r"https?://(github\.com/.*?/.*?/archive/.+\.zip|"
             r"gitlab\.com/.*?/.*?/-/archive/.+\.zip)"
         )
+        self.unstable_uris = re.compile(
+            r"^https?://files.pythonhosted.org/packages/(?:[0-9a-fA-F]{2}/){2}|"
+            r"^https?://patch-diff.githubusercontent.com/raw/.*/pull/[0-9]+.(patch|diff)$"
```

Comment:
I suppose it's no longer "unstable" once PR is merged, though.

Reply:
I think this is fine. There aren't a lot of them in ::gentoo, and maybe there is a way to change a diff for merged PR?

Reply:
I don't think we can rely on it. See also https://github.com/orgs/community/discussions/56522.